llvm/include/llvm-c/DebugInfo.h

```diff
@@ -200,7 +200,29 @@ typedef unsigned LLVMMetadataKind;
 /**
  * An LLVM DWARF type encoding.
  */
-typedef unsigned LLVMDWARFTypeEncoding;
+typedef enum {
```

Comment:
You have LVM -> LLVM everywhere here.

Reply:
Facepalm.

Thankfully, that's easy to fix.

Reply:
Fixed